portal/src/components/roles-and-groups/list/common/ActionButtonCell.tsx

```diff
@@ -2,22 +2,26 @@ import React from "react";
 import BaseCell from "./BaseCell";
 import ActionButton from "../../../../ActionButton";
 import styles from "./ActionButtonCell.module.css";
-import { Text } from "@fluentui/react";
+import { ITheme, Text } from "@fluentui/react";
 import { useSystemConfig } from "../../../../context/SystemConfigContext";
 
 interface ActionButtonCellProps {
   text: string;
   onClick?: (e: any) => void;
+  theme?: ITheme;
```

Comment:
Prefer to expose a prop `variant: "destructive" | "default"` instead of accepting a theme which is difficult to understand how to use it correctly.